pipelines/azure-pipelines.yml

```diff
@@ -40,13 +40,22 @@ steps:
       arguments: '--configuration $(BuildConfiguration) --no-restore'
 
   - task: DotNetCoreCLI@2
-    displayName: 'Publish Website'
+    displayName: 'Publish Squasher App'
     inputs:
       command: publish
       publishWebProjects: false
       projects: 'src/Dfe.Spi.EntitySquasher/Dfe.Spi.EntitySquasher.FunctionApp/Dfe.Spi.EntitySquasher.FunctionApp.csproj'
       arguments: '--configuration $(BuildConfiguration) --output $(Build.ArtifactStagingDirectory)/publish --no-restore --no-build'
 
+  - task: DotNetCoreCLI@2
```

Comment:
Do we need to publish the app? You are running it and publishing the output (which is what I think we do need)

Reply:
I'm actually just running it immediately afterwards to generate a default config. I did this because I wanted to be able to publish the app down the line if we wanted to. It also makes sure it's publish-able.